### DISCUSSION TITLE
Release v0.43.1

### DISCUSSION THREAD

**Discussion by prgres** [terraform-linters/tflint#1625] [Q&A] [answered]
## Introduction
Yesterday, the `--recursive` option was merge to the main branch. On local it works seamless with `brew --HEAD ...` installation but I would like to introduce it to my CI pipelines. 

Could you make a new release with this feature? Ofc I can clone the repo and build the binary but with new release it will be a lot more convenience.

**Comment by wata727** [comment, marked as answer]
We still have work to do for the next release, but we plan to release v0.44 in this month.

**Reply by prgres** [reply]
Great! Glad to hear that. I thought that maybe cherry-picking commit and release as a small update will be easy. But if you plan to do a new version within this month, I can peaceful wait for that

**Reply by wata727** [reply]
v0.44.0 has been released.
https://github.com/terraform-linters/tflint/releases/tag/v0.44.0
